website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -73,10 +73,11 @@ module "orca" {
   ## ORCA Variables
   ## --------------------------
   ## REQUIRED
-  db_admin_password   = var.db_admin_password
-  db_host_endpoint    = var.db_host_endpoint
-  db_user_password    = var.db_user_password
-  orca_default_bucket = var.orca_default_bucket
+  db_admin_password     = var.db_admin_password
+  db_host_endpoint      = var.db_host_endpoint
+  db_user_password      = var.db_user_password
+  orca_default_bucket   = var.orca_default_bucket
+  rds_security_group_id = var.rds_security_group_id
```

Comment:
Shouldn't this (rds_security_group_id) be under the Cumulus Variables section?

Reply:
Moved

Reply:
ok, I was just wondering, I thought I saw it under the Cumulus block in some other sections so was just looking for consistency.